gradle.properties

```diff
@@ -2,6 +2,16 @@ projectVersion=4.0.0-SNAPSHOT
 
 # This prevents the Grails Gradle Plugin from unnecessarily excluding slf4j-simple in the generated POMs
 # https://github.com/grails/grails-gradle-plugin/issues/222
+gradleToolingApiVersion=8.10.2
```

Comment:
I believe you can simply specify `slf4jPreventExclusion = true` now instead of referencing the tooling Api

Reply:
Fixed